main.js

```diff
@@ -0,0 +1,18 @@
+const source = $('#template').html()
+const template = Handlebars.compile(source)
+
+const pepArr = []
+
```

Comment:
You should use clear and understandable names for classes, functions, and variables (for example: firstName etc.) and not something general like "arr", "str" etc.

Reply:
Got it! Thank you, fixed!